### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by Disaer** [CrendKing/avisynth_filter#30]
### Environment
- Filter version: 0.8.2/0.8.3
- AviSynth+ version: v3.6.1
- OS: Windows 10 x64
- Video player: MPC-BE x64 1.5.5.5433, MPC-HC x64 1.9.8
- Splitter filter and video decoder: MPC-BE x64 internal, MPC-HC x64 internal
- Video renderer: madVR, EVR-CP

### Describe the bug
After repeating all install steps from README.md AviSynth Filter doesn't appear in the video player filters list.
AviSynth Filter appears in the video player filters list only after applying activate_remote_control.reg manually.

### To Reproduce
Delete AvsFile and RemoteControl keys under registry path HKEY_CURRENT_USER\SOFTWARE\AviSynthFilter 

**Checklist**
- Try a different **video player** and check if the issue persist. For example, if your main video player is MPC-HC, try MPC-BE.
Findings: tried MPC-HC+madVR and MPC-BE+EVR-CP, issue persist

- Try a different **video renderer** and check if the issue persist. For example, if issue exists with madVR, try EVR.
Findings: doesn't matter

- Try a different **video file** and check if the issue persist. Try another file with different format, dimension and frame rate.
Findings: doesn't matter

[avsf.log](https://github.com/CrendKing/avisynth_filter/files/5649522/avsf.log)

**Comment by CrendKing** [comment]
Oh, I get it now. It is by design. When you don't have avs script, avsf won't do anything other than introducing extra transformation overhead. So by design it disconnects itself from the graph (also see reasoning under [AvsFilterDisconnect()](https://github.com/CrendKing/avisynth_filter#avsfilterdisconnect)). Activating remote control forces it to stay in, as it expects 3rd party application to "injects" an avs script on the fly. You can see these actions in log (the "Disconnect due to AvsFilterDisconnect()" line).

Is there a particular reason you want to see avsf in there even you don't effectively use it?

**Comment by Disaer** [comment]
On my main PC, AVSF was installed by SVP. On my second PC, I've installed AVSF manually to test script problems. So, how should I load a script to it for the first time?

**Comment by CrendKing** [comment]
I'm using MPC-BE as an example. In Options, you register avsf in External Filters. Once registered, remember to set merit to Preferred. Now you can double click on "AviSynth Filter" to bring up a settings page. You can specify your .avs file there.

Also, you can use (portable mode)[https://github.com/CrendKing/avisynth_filter/wiki/Portable-mode] to allow multiple instances of avsf in one computer, if that's not too complicated for you.

Closing. If you have more question, feel free to follow up here.

**Comment by Disaer** [comment]
OK, thanks for the trick with opening settings via double click on "AviSynth Filter".
